controllers/auth.js

```diff
@@ -0,0 +1,86 @@
+const jwt = require( 'jwt-simple' );

```

Comment:
Add a header to each file to say what the file is doing, ie file responsibility.

Reply:
You mean like comment?
// something ?!

Reply:
Yes, a file header comment. Google it